src/thermo/Phase.cpp

```diff
@@ -26,6 +26,7 @@ Phase::Phase() :
     m_xml(new XML_Node("phase")),
     m_id("<phase>"),
     m_temp(0.001),
+    m_elec_temp(0.001),
```

Comment:
As a more generic comment, shouldn't the electron temperature follow the phase temperature in cases that do not involve plasma (non-equilibrium)? This can likely be addressed when implementing Cantera/enhancements#127

Reply:
Yes, it should be.